spec/models/watcher_spec.rb

```diff
@@ -18,4 +20,9 @@
     it { is_expected.not_to allow_value(value).for(:time_window) }
   end
 
+  describe '.conditions' do
+    subject(:conditions) { watcher.conditions }
+
+    it { is_expected.to be_a Hash }
```

Comment:
Should we validate the contents of `watcher.conditions`?

Reply:
No, `Watcher` is the parent class, so I'm testing the bare minimum. The value of `conditions` is being tested for each sub-class:
https://github.com/GSA/search-gov/pull/1031/files#diff-3a6ff73a28d51c2e8ce090c11b93ffe39f22dfa1cae88e8fee148fda1f37e0e8R29
https://github.com/GSA/search-gov/pull/1031/files#diff-c193684fb4eb03a25790862d87b5f5adb7b7b9583894bc03ac9b0fae607f2fa4R30